src/mapGenerator/MapWriter.cpp

```diff
@@ -0,0 +1,284 @@
+// Copyright (c) 2005 - 2015 Settlers Freaks (sf-team at siedler25.org)
+//
+// This file is part of Return To The Roots.
+//
+// Return To The Roots is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 2 of the License, or
+// (at your option) any later version.
+//
+// Return To The Roots is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Return To The Roots. If not, see <http://www.gnu.org/licenses/>.
+
+#include <iostream>
+#include "mapGenerator/MapWriter.h"
+
+bool MapWriter::Write(const std::string& filePath, Map* map)
```

Comment:
Yeah, I've been looking through the code of the "libsiedler2/src" folder but couldn't figure out how to use it. That's why I had a look into the map editor code which was easier to understand. Initially, I actually wanted to use the save game format to make full use the existing code, but couldn't figure out a good way to do it. Is there any specific place (or person) for code related questions?